Libplanet/Net/Messages/Message.cs

```diff
@@ -140,6 +145,11 @@ private enum MessageFrame
         /// </summary>
         public AppProtocolVersion Version { get; set; }
 
+        /// <summary>
+        /// The timestamp of the message is created.
+        /// </summary>
+        public DateTimeOffset Timestamp { get; set; }
```

Comment:
This should be recorded in the changelog.

Reply:
`Message` became public in 0.11, we still need to write changes to changelog?

Reply:
Fair enough then.  👌🏼